### ISSUE TITLE
messages of skipped tests are empty in new HTML report

### ISSUE THREAD

**Issue by lgoltz** [opengeospatial/teamengine#268]
In the old TestNG output the failures if a test was skipped was reported. This may be important to know why the test was skipped. In the new HTML report the message is empty in the overview as well as in the details of a test.

Tested locally (Docker environment) with ets-wfs20-dgiwg 0.3 and TEAM Engine 5.1-SNAPSHOT.

**Comment by keshav-nangare** [comment]
@lgoltz 

Please provide the service URL to reproduce this issue.

**Comment by lgoltz** [comment]
@keshav-nangare http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wfs200?service=WFS&request=GetCapabilities can be used

**Comment by keshav-nangare** [comment]
Thanks Lyn.

The message of some skipped tests are empty in new main HTML page, but some messages are present in TestDetails page.

I have observed both the result i.e. New HTML report as well as TestNG HTML report.

There are only few skipped test which is having the error message that test message are include in the TestDetail html page but missing in the main HTML page(New HTML report).

I have added the fix for it, now the skipped test which is having the message are displayed on new HTML report.

Attached the screenshot here.

![image](https://user-images.githubusercontent.com/14732650/32942640-9cad5a9c-cbaf-11e7-8c94-0e5e4f179465.png)
